pandas/io/formats/style.py

```diff
@@ -879,26 +879,30 @@ def background_gradient(self, cmap='PuBu', low=0, high=0, axis=0,
             1 or 'columns' for columnwise, 0 or 'index' for rowwise
         subset: IndexSlice
             a valid slice for ``data`` to limit the style application to
+        text_color: float or int
+            luminance threshold for determining text color. Facilitates text
+            visibility across varying background colors. From 0 to 1.
+            0 = all text is dark colored, 1 = all text is light colored.
```

Comment:
is this the common name for this field in mpl?

Reply:
IMHO `text_color=0.2` looks very counterintuitive to me. It almost looks like the exact opposite of what this feature is about (not having a constant text color).
Shouldn't the name  at least contain "threshold", e.g. `text_color_threshold`?

Reply:
I agree that `text_color_threshold` is a suitable, is it ok with multiple underscores in the parameter name? I don't think there is a common name for this in mpl (for coloring text in general, they tend to use just `color`, but I believe it would be easy to mistake that for the backgruond color here due to the method's name)